1209/a7ea/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Knixx SW04
+owner: Area3001
```

Comment:
One last thing: Please make sure this has the same capitalisation as the _directory_ the org file is in.